scalableminds/webknossos
# AI-QuickSelect for ND datasets

### Steps to test:
- Connect to SAM server (compare https://scm.slack.com/archives/C5AKLAV0B/p1692289641525739?thread_ts=1692284968.544779&cid=C5AKLAV0B )
- enable in application.conf (features.segmentAnythingEnabled)
- Should be able to use AI Quick Select on timeseries datasets at different time points, result should look sensible

### TODOs:
  - [x] route should take additionalCoordinates 
  - [x] should forward to datastore. fetch correct data
  - [x] send optional basicauth to sam server
  - [x] Send correct additionalCoordinates in `getSamEmbedding`

### Issues:
- fixes #7274
- fixes #7283 

------
